### DISCUSSION TITLE
post_233857

### DISCUSSION THREAD

**Discussion by giscus** [cincuranet/blog-comments#1359] [Comments]
# post_233857

https://www.tabsoverspaces.com/233857-my-new-ac-has-built-in-wifi-and-that-brings-dilemma-into-my-home-automation-hacking

**Comment by ddfamilia** [comment]
Hello! How si this going? Planning to start this journey now, is that remote from the box still in use?

**Reply by cincuranet** [reply]
Nope. Haven't used the remote in last year once. All I need to control, I'm doing through API.

**Reply by ddfamilia** [reply]
Thank you!

**Comment by mkaluza** [comment]
so a small update - or maybe you already know this. If you download a firmware file for your device (https://github.com/maxim-smirnov/gree-wifimodule-firmware) and look for 'known' parameter names, you can find some more (in fact a lot more, but most of them inert). I was for example able to get outdoor temp read, which is useful and also compressor freq and temp, which is just geeky :D

**Reply by cincuranet** [reply]
This is great! Thanks for sharing.
